ui/push-health/Health.jsx

```diff
@@ -37,6 +39,7 @@ export default class Health extends React.PureComponent {
   constructor(props) {
     super(props);
 
+    document.title = 'Push Health';
```

Comment:
You could also use react helmet for that.

Reply:
Maybe even for the icon.

Reply:
Agreed.  Fixed.